packages/flutter_localizations/lib/src/material_localizations.dart

```diff
@@ -14,6 +14,10 @@ import 'l10n/date_localizations.dart' as date_localizations;
 import 'l10n/localizations.dart' show TranslationBundle, translationBundleForLocale;
 import 'widgets_localizations.dart';
 
+// Watch out: the supported locales list in the doc comment below must be kept
+// in sync with the list we test, see test/translations_test.dart, and of course
+// the acutal list of supported locales in _MaterialLocalizationsDelegate.
```

Comment:
Bonus points if you automate this check. Because the list is a plain `List<String>` you could put it in a separate file with a `main()` function that just `print`s it to the console, which can run either in `flutter_tester` or plain ol' Dart VM. You could then do the same with other lists we maintain and compare the output.